ddpui/tests/services/test_kpi_service.py

```
@@ -0,0 +1,393 @@
+            KPIService.get_kpi(99999, org)
+
+    def test_list_kpis(self, org, sample_kpi, seed_db):
+        kpis, total = KPIService.list_kpis(org)
+        assert total >= 1
+        assert any(k.id == sample_kpi.id for k in kpis)
+
+    def test_list_kpis_search_by_name(self, org, sample_kpi, seed_db):
+        kpis, total = KPIService.list_kpis(org, search="Test")
+        assert total >= 1
+
+        kpis, total = KPIService.list_kpis(org, search="nonexistent_xyz")
```

Ruff (0.15.13)

[warning] Unpacked variable `kpis` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)